test/integration/node-specific/feature_flags.test.ts

```diff
@@ -42,4 +45,87 @@ describe('Feature Flags', () => {
       });
     }
   });
+
+  describe('@@mdb.enableMongoLogger', () => {
+    let cachedEnv;
+    const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+
+    before(() => {
+      cachedEnv = process.env;
+    });
+
+    after(() => {
+      process.env = cachedEnv;
+    });
+
+    context('when enabled', () => {
+      context('when logging is enabled for any component', () => {
+        before(() => {
+          process.env.MONGODB_LOG_COMMAND = SeverityLevel.EMERGENCY;
+        });
+
+        it('enables logging for MONGODB_LOG_COMMAND', () => {
```

Comment:
There is nothing wrong with using a specific component as a stand in for all components, but we should make sure to clarify that the point of the test is that the specified component is enabled. Otherwise we're saying that regardless of which component is set, the COMMAND component is enabled.
```suggestion
        it('enables logging for the specified component', () => {
```